api/tools/src/main/java/org/pmiops/workbench/tools/Action.java

```diff
@@ -0,0 +1,10 @@
+package org.pmiops.workbench.tools;
+
+import org.pmiops.workbench.db.WorkbenchDbConfig;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.context.annotation.Import;
+
+// This is a common ancestor that brings in the main database configuration.
+@Configuration
+@Import({WorkbenchDbConfig.class})
+public abstract class Action {}
```

Comment:
nit: what about Tool or CommandLineTool, etc?

Reply:
I was staring at SetAuthority and FetchFireCloudUserProfile and Action is what came to mind. I like Tool.